docs/libraries.md

```diff
@@ -47,6 +47,7 @@ libraries may be outdated due to the fast development cycle of ev3dev.
 
 * Extra languages:
     * [Google Go](https://github.com/ldmberman/GoEV3) updated for ev3dev-jessie by @ldmberman, [original](https://github.com/mattrajca/GoEV3) by @mattrajca
```

Comment:
This should be changed, `s/Google Go/Go/` according to name usage by the Go community.